src/zap/zapinfo.cpp

```diff
@@ -3410,11 +3410,6 @@ void ZapInfo::getReadyToRunHelper(
         }
         break;
 
-    case CORINFO_HELP_READYTORUN_DELEGATE_CTOR:
```

Comment:
It may be better to leave this one alone ... it will save you from fixing up JIT32 in TFS.